src/util/fetch.spec.js

```diff
@@ -0,0 +1,20 @@
+import assert from 'assert';
+import { queryParameters } from './fetch';
+
+describe('queryParameters', () => {
+    it('should generate a query parameter', () => {
+        const data = { foo: 'bar' };
+        assert.equal(queryParameters(data), 'foo=bar');
+    });
+
+    it('should generate multiple query parameters', () => {
+        const data = { foo: 'fooval', bar: 'barval' };
+        const actual = queryParameters(data);
+        assert(['foo=fooval&bar=barval', 'bar=barval&foo=fooval'].includes(actual));
+    });
+
+    it('should generate multiple query parameters with a same name', () => {
+        const data = { foo: ['bar', 'baz'] };
+        assert.equal(queryParameters(data), 'foo=bar&foo=baz');
+    });
```

Comment:
Could you add a test case about URL encoding? The previous function didi it, I want to be sure that `stringify` does it, too.